Misc/NEWS.d/next/Library/2020-01-01-18-44-52.bpo-38871.3EEOLg.rst

```diff
@@ -0,0 +1,3 @@
+Updated the logic of :mod:`lib2to3` when handling filter based statement
```

Comment:
I would suggest the following:

```
Correctly parenthesize filter-based statements that contain lambda expressions in mod:`lib2to3`.
Patch by Dong-hee Na.
```

Reply:
Thanks for the suggestion!! I've updated it!